web/portal/src/main/webapp/WEB-INF/conf/organization/organization-configuration.xml

```diff
@@ -176,6 +176,22 @@
                     </field>
                   </object>
                 </value>
+                <value>
+                  <object type="org.exoplatform.services.organization.OrganizationConfig$Group">
+                    <field name="name">
+                      <string>web-contributors</string>
+                    </field>
+                    <field name="parentId">
+                      <string>/platform</string>
+                    </field>
+                    <field name="description">
+                      <string>the /platform/web-contributors group</string>
+                    </field>
+                    <field name="label">
+                      <string>Web Contributors</string>
```

Comment:
Historically, the label for this group was : Content Management
Should we keep the same ? cc @srenault-meeds 
```suggestion
                      <string>Content Management</string>
```

Reply:
Done as agreed with @srenault-meeds .